api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -458,13 +464,15 @@ class Config:
 
 class GetIndividualOfferWithAddressResponseModel(GetIndividualOfferResponseModel):
     address: AddressResponseIsLinkedToVenueModel | None
-    hasPendingBookings: bool
+    has_pending_bookings: bool
+    is_headline_offer: bool
```

Comment:
idem qu'au dessus, tant pis pour la dette mais gardons l'homogénéité (et idem, tu peux enlever le `if` dans le `getterdict` et l'`alias_generator`